apps/docs/src/guide/cli/commands.md

```diff
@@ -0,0 +1,156 @@
+# Commands
+
+The `fuels` CLI consists of a couple commands.
+
+## `fuels init`
+
+Creates a new `fuels.config.ts` file:
+
+```console
+npx fuels init -w ./sway-programs -o ./src/sway-programs-api
+```
+
+This will give you a minimal configuration:
+
+```ts
+// fuels.config.ts
+import { createConfig } from 'fuels'
+
+export createConfig({
+  workspace: './sway-programs',
+  output: './src/sway-programs-api',
+})
+```
+
+In a nutshell:
+
+```sh
+. # current dir
+├── frontend
+│   ├── fuels.config.ts # your new config
+│   └── src
+│       └── sway-programs-api
+└── sway-programs
+```
```

Comment:
@arboleya Is this directory tree right? This suggests that `sway-programs` is side-by-side with the `frontend`. 

But I reckon that the command `npx fuels init -w ./sway-programs -o ./src/sway-programs-api` should be executed inside the `frontend` dir, right?

Am I missing something here? 

Also, maybe I am doing something wrong on my side, but when I am running `pnpm fuels dev`, I am receiving the following error:

![Screenshot from 2023-10-23 09-18-30](https://github.com/FuelLabs/fuels-ts/assets/30977845/e7af989a-2fdb-43ba-8eaf-09aab3000afe)

Reply:
Good catch. It should be something like this:

```console
. # current dir (NextJS app or similar)
├── sway-programs # Forc's workspace
├── src
│   └── sway-programs-api # Type-safe generated API
├── fuels.config.ts # Your newly generated config file
└── package.json
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1291/commits/325e329f366b4b6a97437742e2675734219ed99e